rmgpy/kinetics/diffusionLimited.py

```diff
@@ -151,6 +152,31 @@ def get_diffusion_limit(self, T, reaction, forward=True):
 
         return k_diff
 
+    def get_henry_law_constant_data(self, spec):
+        solute_data = self.database.get_solute_data(spec)
+        solvent_data = self.solvent_data
+
+        Tmin = solvent_data.get_solvent_coolprop_Tmin()
+        Tcrit = solvent_data.get_solvent_coolprop_Tcrit() 
+        Ts = [float(T) for T in np.linspace(Tmin, Tcrit-0.01, 50)]
+        kHs = [self.database.get_T_dep_solvation_energy_from_LSER_298(solute_data, solvent_data, T)[1]* solvent_data.get_solvent_saturation_pressure(T) for T in Ts]
```

Comment:
It isn't very clear to me how you're calculating kH here can you comment the formula you're using?

Reply:
Done!